.github/workflows/cypress-tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: Cypress tests
+on:
+  pull_request:
+    paths:
+      - 'packages/ui/**/*'
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node-20.5.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
```

Comment:
I found in the docs the following container settings (https://github.com/cypress-io/github-action#docker-image):
```
    container:
      image: cypress/browsers:latest
      options: --user 1001
```

We probably would need to use the latest version, I assume specific containers will always have hardcoded version of Cypress

Reply:
Nice, let's try
```suggestion
    container:
      image: cypress/browsers:latest
      options: --user 1001
```

Reply:
worked without the options actually.